web/client-api/client-api.gradle

```diff
@@ -57,6 +68,89 @@ artifacts {
     }
 }
 
-project.tasks.getByName('quick').dependsOn project.tasks.withType(de.esoco.gwt.gradle.task.GwtCompileTask)
+def gwtUnitTest = tasks.register('gwtUnitTest', Test) { t ->
+    t.systemProperties = [
+            'gwt.args': ["-runStyle HtmlUnit",
+                         "-ea",
+                         "-style PRETTY",
+                         "-war ${layout.buildDirectory.dir('unitTest-war').get().asFile.absolutePath}"
+            ].join(' '),
+            'gwt.persistentunitcachedir': layout.buildDirectory.dir('unitTest-unitCache').get().asFile.absolutePath,
+    ]
+    t.include '**/ClientUnitTestSuite.class'
+    t.useJUnit()
+    t.scanForTestClasses = false
+}
+
+// start a grpc-api server
+String randomSuffix = UUID.randomUUID().toString();
```

Comment:
It would yes, but we are doing this all over for naming docker containers, so that you don't accidentally collide in parallel runs, or separate checkouts, etc. Since this is just for tests, we're not concerned with "oh these tests passed last time, so I'm not even going to run them this time" for integration tests.